lib/house.rb

```diff
@@ -0,0 +1,55 @@
+class House
+  attr_reader :price,
+              :address,
+              :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def price_to_integer
```

Comment:
Keep an eye on the interaction pattern - we wanted this method to be `price`.  Or... we wanted the `price` method to return an integer :)